packages/build/tamagui-build.js

```diff
@@ -622,10 +623,11 @@ async function esbuildWriteIfChanged(
                   //   ? null
                   //   : require.resolve('./babel-plugin-process-env-to-meta'),
                 ].filter(Boolean),
-              }).code
+              })
 
           // output to mjs fully specified
-          await fs.writeFile(mjsOutPath, output, 'utf8')
+          await fs.writeFile(mjsOutPath, output + `\n//# sourceMappingURL=${basename(mjsOutPath)}.map\n`, 'utf8')
```

Comment:
`//# sourceMappingURL=` is needed for VSCode to map to the source code while debugging.

This allows people to set breakpoints and see the execution inside the source file instead of the built (`dist/*`) ones.